src/include/cpu/power/spr.h

```diff
@@ -19,6 +19,7 @@
 #define SPR_HMER_TRIG_FIR_HMI		PPC_BIT(17)
 #define SPR_HMER_HYP_RESOURCE_ERR	PPC_BIT(20)
 #define SPR_HMER_XSCOM_STATUS		PPC_BITMASK(21,23)
+#define SPR_HMER_XSCOM_OCCUPIED		PPC_BIT(23)
```

Comment:
Something is wrong with the spacing

Reply:
GitHub messes up the display, as if plus sign from diff is counted as the first character. It looks fine in normal editor with 8 chars wide tabulations.